ui/frontend/src/components/PersistPage/resourceTemplates.ts

```diff
@@ -9,17 +9,17 @@ import {
 import { ADDRESS_POOL_NAMESPACE } from './constants';
 
 export const ADDRESS_POOL_TEMPLATE = {
-  apiVersion: 'metallb.io/v1alpha1',
-  kind: 'AddressPool',
+  apiVersion: 'metallb.io/v1beta1',
+  kind: 'IPAddressPool',
```

Comment:
Unlike AddressPool IPAddressPool is used only for the ip assignment part. In order to advertise the IP you should create L2Advertisement.

https://github.com/metallb/metallb/blob/main/configsamples/simple_pool_adv_l2.yaml